docs/internals/contributing/bugs-and-features.txt

```diff
@@ -46,13 +46,13 @@ particular:
   include a clear, concise description of the problem, and a set of
   instructions for replicating it. Add as much debug information as you can:
   code snippets, test cases, exception backtraces, screenshots, etc. A nice
-  small test case is the best way to report a bug, as it gives us an easy
-  way to confirm the bug quickly.
+  small test case is the best way to report a bug, as it gives us a
+  helpful way to confirm the bug quickly.
 
-* **Don't** post to |django-developers| just to announce that you have
-  filed a bug report. All the tickets are mailed to another list,
-  |django-updates|, which is tracked by developers and interested
-  community members; we see them as they are filed.
+* **Don't** post to |django-developers| to announce that you have filed a bug
+  report. All the tickets are mailed to another list, |django-updates|, which
```

Comment:
I think something was lost in removing "just". Perhaps substitute with "only". For tickets that need some community debate, it is legitimate to post to django-developers after filing, but the post should be more than "I filed a ticket".